### DISCUSSION TITLE
Is there a guide on the libp2p C++ ?

### DISCUSSION THREAD

**Discussion by xiguamee** [libp2p/cpp-libp2p#236] [Q&A] [answered]
Plz help me. I don't know how to use it.

**Comment by dhuseby** [comment, marked as answer]
Hi @xiguamee, 

There are some examples in the repo: https://github.com/libp2p/cpp-libp2p/tree/master/example but no guide specifically for cpp-libp2p. There is however lots of documentation on core libp2p concepts here: https://docs.libp2p.io/ and here: https://pl-launchpad.io/curriculum/libp2p/objectives/

**Reply by xiguamee** [reply]
Thank you for your help! ❤

**Reply by dhuseby** [reply]
Let me know if you get stuck and need any more help.
